### PR TITLE
Initial submission of annotation-model framework

### PR DESCRIPTION
**Pull request by halindrome**
This is a collection of scripts that the Web Annotation working group is using to author its tests.  There are a couple of example manual tests in the tree to show how the basic flow will work.  The actual tests are being developed separately right now but we would like to migrate into the main WPT repo as soon as possible.

This is my first attempt at adding to WPT, so I expect I did something wrong.  Let me know!

### CODE REVIEW

File: annotation-model/TODO [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Websocket communication to runner tests with a corresponding listener that an
+implementation could use to auto-submit jsonld into the "manual" tests
+
+Change JSONtest logic so that it loads all of the required external files during
+initialization so that it can populate the list of assertions for the HTML in a
+manual test.  Use Promises and Promise.all() to achieve this.
+
+Add a Skip option to tools/runner
+
+Add a "or" mode for collections of assertions.
```
Comment (BigBlueHat) [inline annotation-model/TODO]: @halindrome I believe the Skip logic and possibly the "or" situation are taken care of now, correct?

File: browser-payment-api/OWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+@halindrome
```
Comment (BigBlueHat) [inline browser-payment-api/OWNERS]: @halindrome looks like this snuck in from a different branch? :smiley: :

**Comment by BigBlueHat** [comment]
Other than some very small nits, this looks great. Thanks, @halindrome!

:+1:

File: annotation-model/TODO (hunk 1 above)
Comment (halindrome) [inline annotation-model/TODO]: The skip is something I want to add to runner - right now there is no way to say "I don't want to run this test right now."  

The or mode is supported in the comparisonObject handler.  I will update the TODO file.  Thanks!

File: browser-payment-api/OWNERS (hunk 2 above)
Comment (halindrome) [inline browser-payment-api/OWNERS]: Wow - weird!  Thanks

**Comment by halindrome** [comment]
There was a comment from @tcole3 in a Web Annotation working group meeting that all the .json files in the definitions directory be included into the ajv environment automatically.  This is now implemented via commit https://github.com/w3c/web-platform-tests/pull/3173/commits/e36c68f89a4f7683b20c66fa29d288ee87f49d1e

**Comment by halindrome** [comment]
Git has defeated me whilst trying to squash these commits. I am killing this PR and will do a new one against a rebase'd tree.

**Pull request closed without merging**